sonar-project.properties

```diff
@@ -17,6 +17,8 @@ sonar.tests.inclusions=**/*.test.tsx
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
 # We don't need coverage reports on stories or tests
 sonar.coverage.exclusions=\
-  public/**, cypress/**, src/app/*,\
+  public/**, cypress/**, src/app/*, src/utils/*,\
```

Comment:
Why would we exclude utils? Why exclude app for that matter? Are we not just creating an inaccurate coverage report?